src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -100,6 +101,18 @@ namespace Microsoft.IdentityModel.Tokens
     /// </remarks>
     public delegate string IssuerValidatorUsingConfiguration(string issuer, SecurityToken securityToken, TokenValidationParameters validationParameters, BaseConfiguration configuration);
 
+    /// <summary>
+    /// Definition for IssuerValidatorAsync. Left internal for now while we work out the details of async validation for all delegates.
```

Comment:
>internal [](http://example.com/codeflow?start=50&length=8)

This sounds like a TODO, should we link this with a bug/work item so its in our radar?